__docs__/wonder-blocks-modal/modal-dialog.stories.tsx

```diff
@@ -87,6 +86,7 @@ export default {
             modes: {
                 small: allModes.small,
                 large: allModes.large,
+                thunderblocks: allModes.themeThunderBlocks,
```

Comment:
Now that we include snapshots for thunderblocks too, there are many Chromatic snapshots for modals! Do you think there are things we could do to have fewer? Maybe by combining some stories! (not blocking, but a consideration for later, especially if/when we support other themes!)

Reply:
Agreed! I think that there are some duplicate snapshots between `ModalDialog`, `ModalHeader`, `ModalPanel` stories, I'll see which one of those make sense to remove. We can optimize even more later as you suggested :).